lib/Cake/Test/Case/TestSuite/HtmlCoverageReportTest.php

```diff
@@ -33,7 +33,12 @@ class HtmlCoverageReportTest extends CakeTestCase {
  *
  * @return void
  */
-	public function setUp() {
+	public function setUp() : void {
+		// This test can not be run with PHPUnit 9+.
+		// Because CakeBaseReporter depend on PHPUnit_TextUI_ResultPrinter.
```

Comment:
Similar thing like I mentioned before. Any idea how to decouple from `PHPUnit_TextUI_ResultPrinter`?

Reply:
`HtmlCoverageReport` will no longer be used and could be removed along with `HtmlCoverageReportTest`.